### PR TITLE
Bom 1219

### PR DESCRIPTION
**Pull request by UsamaSadiq**
**Issue:** [BOM-1219](https://openedx.atlassian.net/browse/BOM-1219)
### Description
Removed constraints that cause Django-version conflicts in make upgrade.
Removed python2.7 support from testing.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -108,10 +104,4 @@ passenv =
     PAYPAL_PASSWORD
 deps = -r requirements/e2e.txt
 commands=
-    xvfb-run --server-args="-screen 0, 1600x1200x24" pytest e2e --html=log/html_report.html --junitxml=e2e/xunit.xml
-
-[testenv:py27-e2e]
```
Comment (awais786) [inline tox.ini:113]: don't remove it update it with py3

File: requirements/e2e.in [hunk 2]
```diff
@@ -1,9 +1,9 @@
 -r pins.txt
 
 # Packages required to run e2e tests
-edx-rest-api-client
+edx-rest-api-client<2.0.0   # version>=2.0.0 causes django version conflicts
```
Comment (awais786) [inline requirements/e2e.in]: May be you can add this in pins.txt ?

File: tox.ini (hunk 1 above)
Comment (UsamaSadiq) [reply tox.ini:113]: Python3 e2e tests are already running. The section above it is py35-e2e tests.

File: requirements/e2e.in (hunk 2 above)
Comment (UsamaSadiq) [reply requirements/e2e.in]: moved the restriction to `pins.txt`.

**Comment by UsamaSadiq** [comment]
@jmbowman, could you please have a look at it? All the other ecommerce issues are dependent on this PR.

**Comment by UsamaSadiq** [comment]
Same changes have also been made in PR https://github.com/edx/ecommerce/pull/2764 with updated quality changes.

**Pull request closed without merging**